src/api/Email/emailApi.d.ts

```diff
@@ -1,6 +1,6 @@
 export declare const useSendEmail: () => import("@tanstack/react-query").UseMutationResult<any, Error, {
-    pcolId: string;
     Subject: string;
+    pcolId: string;
```

Comment:
Can't comment on emailAPi.ts directly since it wasn't modified this time around, but it seems like in general you were changing pcolId to be defined as a number, and then using .toString() to make it a string when needed. So this should probably be defined as a number here too.